src/tests/ftest/dfuse/daos_build.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import distro
+import general_utils
+
+from dfuse_test_base import DfuseTestBase
+from command_utils import CommandFailure
+
+class DaosBuild(DfuseTestBase):
+    # pylint: disable=too-many-ancestors,too-few-public-methods
+    """Build DAOS over dfuse
+
+    :avocado: recursive
+    """
+
+    def test_daos_build(self):
+        """Jira ID: DAOS-8937.
+
+        Test Description:
+            This test builds DAOS on a dfuse filesystem.
+        Use cases:
+            Create Pool
+            Create Posix container
+            Mount dfuse
+            Checkout and build DAOS sources.
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm
+        :avocado: tags=daosio,dfuse
+        :avocado: tags=dfusedaosbuild
+        """
+
+        scons = 'scons-3'
+        dist = distro.linux_distribution()
+        if dist[0] == 'CentOS Linux' and dist[1] == '7':
+            self.SkipTest('Newer software distribution needed')
```

Comment:
(pylint-no-member) Instance of 'DaosBuild' has no 'SkipTest' member; maybe 'skipTest'?